koekalenteri-backend/src/handlers/event.ts

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+import "source-map-support/register";
+import { v4 as uuidv4 } from 'uuid';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+const {metricScope} = require("aws-embedded-metrics");
+
+import CustomDynamoClient from "../utils/dynamoClient";
+import { response } from "../utils/response";
+import { metricsSuccess, metricsError } from "../utils/metrics";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
import { metricsSuccess, metricsError } from '../utils/metrics';
```